docs/data/data-grid/faq/faq.md

```diff
@@ -1,156 +0,0 @@
----
-productId: x-data-grid
-title: Frequently Asked Questions - React Data Grid
-githubLabel: 'component: data grid'
-packageName: '@mui/x-data-grid'
----
-
-# Frequently asked questions
-
-<p class="description">Can't find what you are looking for? The FAQ section has answers to some of the most frequent questions and challenges.</p>
-
-If you still have trouble, you can refer to the [support page](/x/introduction/support/).
-
-## How to pass props to the Data Grid for better performance?
```

Comment:
Nice to see the FAQ size shrunk. FAQs are usually about giving up on finding a great docs architecture to place the information in the right place.

I don't know if we need to kill thoughts because they are things that we don't want people to be able to find (effective wherever is in the FAQ), unless they do an explicit search for it. I don't think to consider for the future if we find a use case.

Reply:
> FAQs are usually about giving up on finding a great docs architecture to place the information in the right place.

This is how the MUI X FAQs felt to me, which is why I made it a priority to relocate all of that content. I would honestly be surprised if we could find a worthwhile use case for this page in the future. I think I would always prefer to find (or create) a more specific place for any new content. As technical writers, we should be able to anticipate the questions that users will have and address them in the same docs where those questions might arise, rather than gathering unsorted answers to random questions in a catch-all FAQ doc. Not to say there's never a use for FAQs, I just don't really see the utility in this context.